tests/microos/install_image.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install SL Micro image on bare metal disk
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+use Utils::Backends;
+use Utils::Architectures 'is_x86_64';
+use power_action_utils 'power_action';
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+
+    # Use image name from HDD_1 variable
+    my $image = get_required_var('HDD_1');
+    # Use sda as target disk by default
+    # If we want use nvme disks in the future, we will have to extend bellow logic with custom variable and new partition naming shceme
+    my $device = "/dev/sda";
```

Comment:
Maybe it's enough for a start to do what you actually describe above, only use "sda by default"

```suggestion
    my $device = get_var('MICROOS_INSTALL_IMAGE_TARGET_DEVICE', '/dev/sda');
```

Reply:
i had exactly that line, but didn't want introduce another just from the beginng :) will think about it

Reply:
I rather added support now https://openqa.suse.de/tests/15539169#step/install_image/2 .